monai/networks/utils.py

```diff
@@ -615,6 +626,101 @@ def convert_to_torchscript(
     return script_module
 
 
+def convert_to_trt(
+    model: nn.Module,
+    precision: str,
+    inputs_shape: Sequence[Sequence[int]],
+    dynamic_batchsize: Sequence[Sequence[int]],
+    ir: str,
+    verify: bool = False,
+    device: torch.device | None = None,
+    rtol: float = 1e-4,
+    atol: float = 0.0,
+):
+    """
+    Utility to convert a model into TorchScript model and save to file,
+    with optional input / output data verification.
+
+    Args:
+        model: source PyTorch model to save.
+        precision: the weight precision of converted TensorRT engine based torchscript models. Should be 'fp32' or 'fp16'.
+        inputs_shape: the inputs' shape that is used to generate random input during the convert. Should be a list with elements
+            like [N, C, H, W] or [N, C, H, W, D].
+        dynamic_batchsize: a list contains three number list elements to define the batch size range of inputs for the
+            converted model. In each element, the three number means `MIN_BATCH, OPT_BATCH, MAX_BATCH` in order.
+        ir: the intermediate representation way to transform a pytorch module to a TensorRT engine based torchscript. Could
+            be choose from `(script, trace)`.
+        verify: whether to verify the input and output of TorchScript model.
+            if `filename_or_obj` is not None, load the saved TorchScript model and verify.
```

Comment:
The docstring is not correct, should be TensorRT instead of TorchScript.
Please double-confirm all the other doc-strings.

Thanks.

Reply:
Thanks, will update this part.